templates/README.md

```diff
@@ -0,0 +1,38 @@
+# How to setup AWS Credentials rotation
+
+These files have to be deployed in the platform namespace.
+
+- prerequisite
+
+  ```text
+  - you have access to kubernetes cluster hosting 'platform' namespace.
+  - docker image 'nuxeo/aws-iam-credential-rotate' is available.
+  - create or retrieve credentials from aws:
+    - Access key ID
+    - Secret access key
+  ```
+
+## Create Secret
+
+[secret](secret.yaml) file will be created under the 'platform' namespace:
+
+- aws-iam-user-credentials secret
+
+```bash
+    > $ kubectl create -f templates/secret.yaml
```

Comment:
Code example in this repo doesn't prepend shell prompt. I'm in favor to be consistent with existing README.

I don't have strong thoughts about prepending a character or not, if we choose to do it, I'm in favor of `$`.

Reply:
thanks,  will be removed